cfpb/consumerfinance.gov
# Wagtail: Add aria-label option to links

Pages like https://www.consumerfinance.gov/owning-a-home/ have links like "Learn more" that are not descriptive enough. This PR adds the ability to add an aria-label to links in Wagtail that aren't descriptive enough for assistive devices.


## Additions

- Adds aria-label input field to Wagtail hyperlinks.


## How to test this PR

1. Visit a page with a hyperlink like Buying a House (http://localhost:8000/admin/pages/3701/edit/) and see that you can add an aria label to link.


## Screenshots

<img width="912" alt="Screen Shot 2021-06-22 at 1 05 49 PM" src="https://user-images.githubusercontent.com/704760/122969424-a1ccc100-d35a-11eb-9e4d-0a25bee05bea.png">
